server/src/test/java/org/apache/cloudstack/networkoffering/CreateNetworkOfferingTest.java

```diff
@@ -250,4 +257,12 @@ public void createVpcNtwkOffWithNetscaler() {
         assertNotNull("Vpc Isolated network offering with Vpc and Netscaler provider ", off);
     }
 
+    @Test
+    public void createVpcNtwkOffWithEmptyDisplayText() {
+        createNetworkOfferingCmd = new CreateNetworkOfferingCmd();
+        String netName = "network";
+        ReflectionTestUtils.setField(createNetworkOfferingCmd, "networkOfferingName", netName);
+        Assert.assertEquals(createNetworkOfferingCmd.getDisplayText(), netName);
+    }
```

Comment:
should this be in a file called `CreateNetworkOfferingCmdTest` instead of here?

Reply:
Fixed